### DISCUSSION TITLE
第十一讲 大时代何去何从 小人物应对有道 - 课堂笔记

### DISCUSSION THREAD

**Discussion by guttasingapore** [RoadSideUniversity/ClubHouse2024#30] [General]
正值中美APEC会谈， 美国幸运的两个黄金期， 我们这代人何德何能，幸运的见证历史，两个冷战间的黄金期已经过去了

**Comment by guttasingapore** [comment]
为什么只有美国能够凝聚文明智慧和精英，难道只有美国能够做到这点吗？
后方优势与后发劣势， 一体两面难以抄袭， 制度结构的巨大障碍 （杨小凯）， 实践和理论无法有效融合（林毅夫）， 中国学者或专家整个群体基本上没有独立人格， 影响力受其立场限制。 东方大国的结构和演变，（王沪宁对东方大国过去20年的巨大影响力）
《美国反对美国》王沪宁 https://github.com/zealotCE/America-Against-America/tree/master

**Comment by guttasingapore** [comment]
民主的价值表达在自由思想，起码需要保证有不同声音，求同与存异的辩证统一 （这个是奢望吗？）
习拜在APEC的会面应该是构造底线思维，军事议题为主，不需要过多期望过分解读， 双方都在拖时间，比较悲观
案例1： 李白兄问了3个GPT/open AI， 唯一能够提供有效答案的是Google

**Comment by guttasingapore** [comment]
真相： 个人会主观选择自己希望听到的真相， 历史没有绝对的真相
坦白： 李白推荐顾准，李锐很坦白
https://github.com/dshenwenxi/1667/blob/master/%E6%9D%8E%E9%94%90%E5%8F%A3%E8%BF%B0%E5%BE%80%E4%BA%8B_17067546_1809105.mobi

过去一年科技发展太快了，快速迭代涌现
我们这个时代与父辈的同与不同，我们有没有对上一代的反省能力，有没有对下一代的影响能力？
传承与知识：做事做人， 现在我们的知识体系能够帮助我们跟上科技的进步并且指导自己的孩子们 （thanks hengsir input)
现在我们的孩子还会问我们问题吗? 价值判断往往会被抛弃，往往孩子们会主动问我们关于社会问题
XP同学主动帮助孩子写作业， 赞！
L：孩子主动问的社会问题，关于选择生育是自私还是无私，人性本性就是自私的？如果没有个人主义的独立思考，可能会比较值得警惕。父母不要扮演伟光正，真诚表达个人意见比较可取。
李白：国富论 亚当·斯密的人性观 关于人性自私的论点
参考薛兆丰 https://bootsoon.github.io/economics/2017/02/28/006.html
亚当·斯密的《国富论》（ Wealth of Nation，1776 ）“每一个人，不需要自己关心社会福利，他也不知道自己怎么去推动社会的福利。他只需要关心自己，追求他自己的福利就可以了。但是他在追求自己福利的过程中，会有一只看不见的手，让他的努力转变为对公共事业的推动。这只看不见的手，会让他的自私自利推动社会福利的改进。”

**Comment by guttasingapore** [comment]
XP:小孩子做了充分准备来挑战父辈是很有意思的
Piers Morgan的关于时政的辩论值得参考
https://github.com/orgs/RoadSideUniversity/discussions/25#discussioncomment-7550002
美国大选投票率：2016 美国总统选举  https://en.wikipedia.org/wiki/2016_United_States_presidential_election

**Reply by guttasingapore** [reply]
美国大选投票率中间选民的5%如果被媒体操控将影响大选结果，小人物能做什么？

**Reply by guttasingapore** [reply]
关于#台湾#沈伯阳#认知战
https://zh.wikipedia.org/zh-sg/沈伯阳
https://www.rfa.org/mandarin/yataibaodao/gangtai/hx-07272023111552.html
https://www.cna.com.tw/news/aipl/202309110006.aspx

**Reply by guttasingapore** [reply]
thanks libai sharing: https://www.youtube.com/watch?v=hvz1I2UiqFI

**Comment by guttasingapore** [comment]
真诚： 战胜自己，从“说到做到”开始是非常好的出发点，需要有足够的自我认知，有勇气有能力挑战自己。宠爱自己也可以进步。 
如果吃一天的苦能得到99天的幸福，你愿意吗？对别人要求你吃的苦，一定要保持高度警惕
需要个人自主的选择，需要有足够的自我反省和认知，如果一天的苦都不需要吃就能得到幸福，为什么不？
多巴胺，内啡肽，荷尔蒙的讨论，值得深入探讨

**Reply by guttasingapore** [reply]
ZT 从快感到成就感：多巴胺vs内啡肽 https://zhuanlan.zhihu.com/p/24697188

**Reply by guttasingapore** [reply]
疼痛， 应急反应，安慰剂，代偿，人体自我分泌的激素，类似抗体，建议通过日常生活以自然方式取得激素平衡. 能用心理暗示达到效果的就不需要药品和外科手术， 色彩在医疗上的应用 thanks Yolanda

**Comment by ergatea** [comment]
https://youtu.be/fBbqTKlCXOU?si=lIcmC1TCGkr0E39j 听听林毅夫怎么回答的。

Gutta ***@***.***> 于2023年11月15日周三 12:43写道：

**Reply by guttasingapore** [reply]
疫情後中國經濟未来走向的发言，林毅夫分析在複雜的國際形勢下中國的機遇與挑戰（Sep 12, 2023)。感谢XP同学推荐，展开到苏南的微观现实。

**Reply by guttasingapore** [reply]
参考文章 Will China Continue to Grow? Weijian Shan, November 14, 2023
[Will China Continue to Grow.pdf](https://github.com/RoadSideUniversity/ClubHouse2023Nov/files/13361578/Will.China.Continue.to.Grow.pdf)

**Reply by Lewisrxliu** [reply]
- 延续这个话题，回顾历史，参考另一篇新鲜的”从Burlingame到Filoli ---拜习会随感” 关于蒲安臣

**Reply by Lewisrxliu** [reply]
2023.11.18 VOA专访 Robert Daly

专访戴博：中国崛起进程脆断，何日再现复兴？

https://www.voachinese.com/a/7359894.html
https://youtu.be/VDOM8ET-NDw?si=9xcKaQ5H2hCclG1F

**Reply by Lewisrxliu** [reply]
[Bai Xi Meet- from Burlingame to Filoli.pdf](https://github.com/RoadSideUniversity/ClubHouse2023Nov/files/13401971/Bai.Xi.Meet-.from.Burlingame.to.Filoli.pdf)

**Comment by Lewisrxliu** [comment]
https://www.voachinese.com/amp/risch-colleagues-biden-xi-meeting-should-bring-concrete-deliverables-not-more-concessions-20231114/7354928.html

https://www.zaobao.com.sg/realtime/china/story20231115-1450078

http://www.news.cn/2023-11/15/c_1129976165.htm

**Reply by Lewisrxliu** [reply]
https://youtu.be/IE5tsowbCDw?si=DvdtFmHX-7gZVDPj

**Comment by yibingbingwu** [comment]
昨晚最后谈到美国军工企业浪费的问题，找到了这篇文章，当时兴盛一时：[**空军最新订单：每个马桶盖一万四千美元**](https://www.nytimes.com/2018/12/19/opinion/pentagon-budget-military-spending-waste.html)

号称下订单的人还在Reddit上开了个[“实话实说”节目](https://www.reddit.com/r/IAmA/comments/kfhlq/iama_active_duty_military_guy_who_buys_10000/)
